crates/cache/src/worker.rs

```diff
@@ -5,6 +5,14 @@
 //! but we guarantee eventual consistency and fault tolerancy.
 //! Background tasks can be CPU intensive, but the worker thread has low priority.
 
+#![cfg_attr(
+    not(test),
+    expect(
+        clippy::useless_conversion,
+        reason = "test/not have a different definition of `SystemTime`",
```

Comment:
?

Reply:
What I meant (and should have written out) is that the "test/not" part of the `reason` message was hard to parse.